server/src/main/java/org/opensearch/cluster/metadata/IndexMetadata.java

```diff
@@ -282,6 +282,16 @@ public Iterator<Setting<?>> settings() {
         Property.IndexScope
     );
 
+    /**
+     * Used to specify SEGMENT replication type as default for all indices. By default, this is false.
+     */
+    public static final Setting<Boolean> DEFAULT_REPLICATION_TYPE_SETTING_SEGMENT = Setting.boolSetting(
```

Comment:
Could be renamed to DEFAULT_SEGMENT_REPLICATION_TYPE_CLUSTER_SETTING or CLUSTER_DEFAULT_SEGMENT_REPLICATION_TYPE_SETTING to make it obvious this is a cluster level setting.